test/support.js

```diff
@@ -1 +1 @@
-import '@agoric/synpress/support/index';
```

Comment:
reason for changing this? `index should work just fine`

Reply:
I have customized the `support.js` file in `@agoric/synpress`. In the `before` hook, we originally set up the wallet.  That sets the wallet every time I visit a different origin. I found this approach to be excessive, so I opted to override it. Now, I configure the wallet setup explicitly within a specific test to ensure we don't set the wallet everytime we visit a different origin.

Reply:
I was wondering if we should remove setting up wallet in the `before` hook?